packages/web-react/src/components/Card/stories/CardArtwork.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { AlignmentX } from '../../../constants';
+import { Button } from '../../Button';
+import { Container } from '../../Container';
+import { Icon } from '../../Icon';
+import Card from '../Card';
+import CardArtwork from '../CardArtwork';
+import CardBody from '../CardBody';
+import CardEyebrow from '../CardEyebrow';
+import CardFooter from '../CardFooter';
+import CardTitle from '../CardTitle';
+import ReadMe from '../README.md';
+
+const meta: Meta<typeof CardArtwork> = {
+  title: 'Components/Card',
+  component: CardArtwork,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    alignmentX: {
+      control: 'select',
+      description: 'Alignment inside CardArtwork component.',
+      options: [...Object.values(AlignmentX)],
+      table: {
+        defaultValue: { summary: AlignmentX.LEFT },
+      },
+    },
+  },
+  args: {
+    alignmentX: 'left',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof CardArtwork>;
+
+export const CardArtworkComponent: Story = {
```

Comment:
```suggestion
export const CardArtworkPlayground: Story = {
```

so the URL is the same as elsewhere